dnsimple/registrar.go

```diff
@@ -94,10 +108,23 @@ type DomainTransferRequest struct {
 	EnableAutoRenewal bool `json:"auto_renew,omitempty"`
 }
 
+// DomainTransfer represents the result of a domain renewal call.
+type DomainTransfer struct {
+	ID           int    `json:"id"`
+	DomainID     int    `json:"domain_id"`
+	RegistrantID int    `json:"registrant_id"`
+	State        string `json:"state"`
+	AutoRenew    bool   `json:"auto_renew"`
+	PrivateWhois bool   `json:"private_whois"`
+	PremiumPrice int    `json:"premium_price"`
```

Comment:
@weppos similar to https://github.com/dnsimple/dnsimple-go/pull/52/files#r91865332